src/e2e-test/features/recordsplitter/FileToRecordSplitter.feature

```diff
@@ -0,0 +1,110 @@
+@RecordSplitter
+Feature: RecordSplitter transform - Verify File source data transfer using RecordSplitter transform
```

Comment:
these test cases are already covered by previous test cases, can be removed

Reply:
Removed.